.monorepolint.config.ts

```diff
@@ -32,6 +32,78 @@ glob.sync(path.join(__dirname, "packages", "turf-*")).forEach(pk => {
 
 module.exports = {
   rules: {
-    ":alphabetical-scripts": {}
+    ":alphabetical-scripts": {},
+
+    ":package-entry": [
+      {
+        options: {
+          entries: {
+            main: "dist/js/index.js",
+            types: "dist/js/index.d.ts",
+            files: ["dist"]
+          }
+        },
+        includePackages: TS_PACKAGES
+      },
```

Comment:
So this first chunk targets the TS-based packages?

Is there any reason we're doing `dist/js/index.js` rather than `dist/index.js`?

Reply:
Yeah it gives us room for dist/es/* for the modules later.